src/stiefel.jl

```diff
@@ -167,24 +167,18 @@ function stiefelexp(W::AbstractTensorMap,
                     A::AbstractTensorMap,
                     Z::AbstractTensorMap,
                     α::Real)
-    S = spacetype(W)
-    G = sectortype(W)
-    dims = TensorKit.SectorDict{G,Int}()
-    generator = Iterators.map(blocks(W)) do (c, b)
+    V = fuse(domain(W))
```

Comment:
That's clever 😃 : using functions what they are meant for instead of reimplementing them 👍

Reply:
I have my moments 😁